wd_comp.c

```diff
@@ -778,6 +778,7 @@ static const struct wd_alg_ops wd_comp_ops = {
 
 int wd_comp_env_init(void)
 {
+	wd_comp_env_config.disable_env = 0;
```

Comment:
What's the meaning of "disable_env" at here? Why do you need to clear it and set it again?

Reply:
I use it  to split 2 scenarios here, 
if (config->disable_env)
    var_s = var->def_val;//If we use wd_alg_ctx_num ,we do not want system env influence
else
    var_s = getenv(var->name);//get value from system env